set.mm

```diff
@@ -22293,8 +22293,8 @@ the definition (which used to be ~ eu6 , while this definition was then
   ${
     $d x y $.  $d ph y $.
     $( An alternate way to express existential uniqueness.  (Contributed by NM,
-       8-Jul-1994.)  Add a disjoint variable condition on ` ph , y ` .
-       (Revised by Wolf Lammen, 29-May-2019.) $)
+       8-Jul-1994.)  Convert a non-free premise to a disjoint variable
+       condition on ` ph , y ` .  (Revised by Wolf Lammen, 29-May-2019.) $)
```

Comment:
I would prefer:
> Replace a nonfreeness hypothesis with a disjoint variable condition on ` ph , y ` to reduce axiom usage.

Ideed: 1) hypothesis is our preferred term for $e-statements  2) the hypothesis is not "nonfree", which makes no sense: it is a hypothesis of "nonfreeness" 3) we explain why we did that: "to reduce axiom usage".

Rk: I would still like the version with the nonfreeness hypothesis !

Reply:
comment is updated

Reply:
> By the way, what does rk stand for? Roundhouse kick?

:laughing: I meant "remark", but I have no idea how much it's used; maybe "rmk" or "rem" is better, but then "remark" is not much longer anyway so I could type it in full !